src/utils/test/util.spec.ts

```diff
@@ -155,10 +155,16 @@ describe('util', () => {
       // improperly formatted JSON - note the lack of ':'
       const diagnostic = util.parsePackageJson('{ "someJson" "value"}', mockPackageJsonPath);
 
+      // due to missing colon in input
+      // the text of this message was changed in node version 20
+      const messageText = Number(process.versions.node.split(".")[0]) > 20
+        ? "Unexpected string in JSON at position 13"
+        : "Expected ':' after property name in JSON at position 13";
+
```

Comment:
this is def weird, do we have an existing pattern for doing this / is there a better way?

The problem is just that the text of this error message is different in node 20 than it is in 14, 16, and 18.

Reply:
Yeah, this is definitely weird. IDK if we have a better pattern/prior art for dealing with it.

Alternative to the ternary here, maybe we re-assess the value of the message itself (in part or as a whole) in this test. Maybe we consider asserting that this is _some_ string ending with "in JSON at position 13" or just a string as a whole?

Reply:
I would probably just check that is identifies the correct location/position

Reply:
that sounds reasonable to me!